test/e2e/storage/pvc_protection.go

```diff
@@ -48,7 +48,10 @@ var _ = utils.SIGDescribe("PVC Protection", func() {
 
 		By("Creating a PVC")
 		suffix := "pvc-protection"
-		defaultSC := getDefaultStorageClassName(client)
+		defaultSC, _ := getDefaultStorageClassName(client)
+		if defaultSC == "" {
+			framework.Skipf("no default storagclass configured skipping")
```

Comment:
minor typo in `storagclass`
-> "storage class"

Reply:
There's a similar pr out here: https://github.com/kubernetes/kubernetes/pull/68949

I think skipping is fine for now, although we may want to think of a better way for how we define/skip tests. Maybe via tags?  By blindly skipping, providers won't know if they've misconfigured their environment and may be missing out on test coverage.

Reply:
Yeah I think we may want to tag all the tests that depend on default SC. We have to do full sweep of test suite. 

Also #68949 is although related but does not affects at least in baremetal environments because there is no default SC anyways installed.